1.1/spec/11-Part-08.adoc

```diff
@@ -481,3 +481,100 @@ geof:getSRID (geom: ogc:geomLiteral): xsd:anyURI
 
 Returns the spatial reference system URI for `geom`.
 
+===== Function: geof:maxX
+
+```
+geof:maxX (geom: ogc:geomLiteral): xsd:double
+```
+
+Returns the maximum X coordinate for `geom`.
+
+===== Function: geof:maxY
+
+```
+geof:maxY (geom: ogc:geomLiteral): xsd:double
+```
+
+Returns the maximum Y coordinate for `geom`.
+
+===== Function: geof:maxZ
+
+```
+geof:maxZ (geom: ogc:geomLiteral): xsd:double
+```
+
+Returns the maximum Z coordinate for `geom`.
+
+===== Function: geof:minX
+
+```
+geof:minX (geom: ogc:geomLiteral): xsd:double
+```
+
+Returns the minimum X coordinate for `geom`.
+
+===== Function: geof:minY
+
+```
+geof:minY (geom: ogc:geomLiteral): xsd:double
+```
+
+Returns the minimum Y coordinate for `geom`.
+
+===== Function: geof:minZ
+
+```
+geof:minZ (geom: ogc:geomLiteral): xsd:double
+```
+
+Returns the minimum Z coordinate for `geom`.
+
+=== Spatial Aggregate Extension
+
+This clause establishes the Spatial Aggregate extension requirements class with URI `/req/spatial-aggregate-extension`.
+Spatial Aggregate Functions may be used in the SELECT clause of a SPARQL query and in the WHERE clause.
```

Comment:
I'm having a hard time seeing where aggregates can be used in a WHERE clause except in nested subqueries. Should we say "used in the SELECT, HAVING and ORDER BY clauses of a SPARQL query"

Reply:
Sorry, my bad. The WHERE was a remnant of when I first wrote this paragraph with also the minX etc. functions in mind.
I think SELECT, HAVING and ORDER BY seems reasonable to me. 
I added it to the draft.